zephyr/esp32/CMakeLists.txt

```diff
@@ -4,7 +4,9 @@ if(CONFIG_SOC_SERIES_ESP32)
 
   zephyr_compile_options(-fstrict-volatile-bitfields)
   zephyr_compile_definitions_ifndef(CONFIG_MCUBOOT CONFIG_APP_BUILD_USE_FLASH_SECTIONS)
-  zephyr_compile_definitions_ifndef(CONFIG_MCUBOOT BOOTLOADER_BUILD=y) # See Zephyr-ESP / ZEP-820
+
+  zephyr_compile_definitions_ifdef(CONFIG_MCUBOOT BOOTLOADER_BUILD) # See Zephyr-ESP / ZEP-820
```

Comment:
Should we put this kind of comment here?

Reply:
fixed